modules/veteran_verification/SERVICE_HISTORY.yml

```diff
@@ -0,0 +1,142 @@
+
+    ### Authorization
+
+    API requests are authorized using a Bearer token issued through an OpenID
+    Connect service to allow third-party applications. The token should be
+    submitted as an `Authorization` header in the form `Bearer <toekn>`.
+
+    ### Service History Request
+
+    Allows a third-party application to request service history with the authorization
+    of a Veteran.
+
+    1. Client Request: GET https://api.vets.gov/services/veteran_verification/v0/service_history
+       * Provide the Bearer token as a header: `Authorization: Bearer <token>`
+
+    2. Service Response: A JSON API object with the Veteran's complete service history
+
+    ## Reference
+
+    Raw Open API Spec: http://dev-api.vets.gov/services/appeals/docs/v0/api
+
+  termsOfService: ''
+  contact:
+    name: Vets.gov
+tags:
+  - name: service_history
+    description: Veteran Verification - Service History
+servers:
+  - url: https://dev-api.vets.gov/services/veteran_verification/{version}
+    description: Vets.gov API development environment
+    variables:
+      version:
+        default: v0
+paths:
+  /service_history:
+    get:
+      tags:
+        - service_history
+      summary: Retrieve service history of authorized Veteran
+      operationId: getServiceHistory
+      security:
+        - bearer_token: []
+      responses:
+        '200':
+          description: Service History retrieved successfully
+          content:
+            application/json:
+              schema:
+                required:
+                  - data
+                properties:
+                  data:
+                    $ref: '#/components/schemas/ServiceHistoryEpisode'
+        '404':
+          description: No service history found
+components:
+  securitySchemes:
+    bearer_token:
+      type: http
+      scheme: bearer
+  schemas:
+    ServiceHistoryEpisode:
+      description: |
+        Service History for authorized Veteran
+      type: object
+      properties:
+        id:
+          type: string
+          description: Service History Episode ID
+          example: "12312AASDf"
+        type:
+          type: string
+          example: service-history-episodes
+        attributes:
+          type: object
+          properties:
+            start_date:
+              type: string
+              format: date
+              description: start date of a service history episode (YYYY-mm-dd)
+              example: '1948-04-08'
+            end_date:
+              type: string
+              format: date
+              description: end date of a service history episode (YYYY-mm-dd)
+              example: '1950-05-10'
+            branch:
+              type: string
+              description: Branch of military including National Guard or Reserve status
+              example: "Air Force"
+            discharge_status:
+              type: string
+              description: status level of discharge from service history episode
+              enum:
```

Comment:
Should document all the possible values in the eMIS data dictionary. For the "for VA purposes" values, we should add a note somewhere (either inline here or in the overall description) that such statuses were administratively changed and were originally either "Bad Conduct" or  "Other Than Honorable". That goes for both J and K values.

Reply:
How about:

Character of discharge from service episode. These are derived from the following eMIS status codes. A becomes 'honorable'. B is 'general'. D is 'bad-conduct'. F is 'dishonorable'. J was originally a 'bad-conduct' or 'dishonorable', but has been changed to 'honorable' (as reported by this API) via an administrative decision. K was originally a 'bad-conduct' or 'dishonorable' and was determined to still be after an administrative hearing (reported as 'dishonorable' by this API).

Reply:
The full list is:
```
Under honorable conditions (general)
Bad conduct
Under other than honorable conditions
Dishonorable
Honorable (absence of a negative report)
Honorable for VA Purposes (Administrative values used only in VA Satellites)
Dishonorable for VA Purposes (Administrative values used only in VA Satellites)
Uncharacterized
Unknown
```

I would not refer to the eMIS letter codes, I feel like they are a remnant of old byte-miserly system integration. Let's go with this, and get feedback:

Character of discharge from service episode. Possible values are:
`honorable, general, bad-conduct, other-than-honorable, dishonorable, honorable-absence-of-negative-report, honorable-for-va-purposes, dishonorable-for-va-purposes, uncharacterized, unknown`

Both "honorable-for-va-purposes" and "dishonorable-for-va-purposes" represent a change in character of discharge based on an administrative decision, for purposes of VA benefits administration. The original character of discharge for other purposes was either "bad-conduct" or "other-than-honorable". "honorable-absence-of-negative-report" represents a an unreported character of service that DoD classifies as honorable. 


Under honorable conditions (general)
Bad conduct
Under other than honorable conditions
Dishonorable
Honorable (absence of a negative report)
Honorable for VA Purposes (Administrative values used only in VA Satellites)
Dishonorable for VA Purposes (Administrative values used only in VA Satellites)
Uncharacterized
Unknown

Reply:
Understood. I think I was a little confused from the slack discussion, but can do this.